metricshub-doc/src/site/markdown/configuration/configure-agent.md

```diff
@@ -301,6 +301,49 @@ resourceGroups:
             timeout: 120s
 ```
 
+#### SNMP V3
+
+Use the parameters below to configure the SNMP V3 protocol:
+
+| Parameter        | Description                                                                    					   |
+| ---------------- | ----------------------------------------------------------------------------------------------------- |
+| snmpv3           | Protocol used to access the host using SNMP v3.                               		    			   |
+| timeout          | How long until the SNMP request times out (Default: 120s).                     					   |
+| version          | The version of the SNMP protocol (v3).                                 	   		    			   |
+| community        | The SNMP Community string (Default: public). 														   |
+| port             | The SNMP port number used to perform SNMP V3 queries (Default: 161).           					   |
+| contextName      | The name of the SNMP v3 context, used to identify the collection of management information.		   |                                              
+| authType         | The SNMP v3 authentication protocol (MD5, SHA or NoAuth) to ensure message authenticity.    		   |
+| privacy          | The SNMP v3 privacy protocol (DES, AES or NO_ENCRYPTION) used to encrypt messages for confidentiality.| 

```

Comment:
`The SNMP v3 privacy protocol (DES, AES or NONE) used to encrypt messages for confidentiality.`